internal/storage/disk/disk.go

```diff
@@ -14,7 +14,7 @@ import (
 	"sync/atomic"
 	"time"
 
-	"github.com/dgraph-io/badger/v2"
+	"github.com/dgraph-io/badger/v3"
```

Comment:
Will it auto upgrade a v2 database on opening?

Reply:
I'm not sure, haven't tried. I would also like @qiaowei-g to review the options. There's many changes in badger and apparently they are using manual memory allocation to use a bit less memory and free it better.

Reply:
based on the description in [badger-#1555](https://github.com/dgraph-io/badger/pull/1555), v2 data cannot be read by v3, so we will have to handle the deployment carefully to ensure no data loss.

Reply:
Alright, I'll mark it as 1.3 release so we don't mix things up.